config/routes.rb

```diff
@@ -3,4 +3,6 @@
   devise_for :users
   root 'welcome#index'
 
+  resources :registered_applications
```

Comment:
To avoid exposing unused, non-existent routes, consider limiting the routes generated using the `only` option:

```ruby
# Generates 7 routes, including 3 unused routes for update, edit, show:
resources :registered_applications

# Favor generating only needed routes:
resources :registered_applications, only: [:index, :create, :destroy, :new]
```